content/influxdb/cloud-dedicated/guides/api-compatibility/v1/_index.md

```diff
@@ -457,7 +457,8 @@ For {{% product-name %}} v1 API `/query` requests, set parameters as listed in t
 
 Parameter | Allowed in | Ignored | Value
 ----------|------------|---------|-------------------------------------------------------------------------
-`chunked` |            | Ignored | N/A _(Note that an unbounded query might return a large amount of data)_
+`chunked` | Query string | Honored | Returns points in streamed batches instead of in a single response. If set to `true`, InfluxDB chunks responses by series or by every 10,000 points, whichever occurs first.
+`chunked_size` | Query string | Honored | **Requires `chunked` to be set as well**. If set to a specific value, InfluxDB chunks responses by series or by that number of points.
```

Comment:
We also need to add these in API reference docs:
- https://docs.influxdata.com/influxdb/cloud-serverless/api/v1-compatibility
- https://docs.influxdata.com/influxdb/cloud-dedicated/api/v1-compatibility
- https://docs.influxdata.com/influxdb/clustered/api/v1-compatibility